gammapy/makers/tests/test_reduce.py

```diff
@@ -279,6 +331,19 @@ def test_datasets_maker_spectrum(observations_hess, makers_spectrum, spectrum_da
     assert_allclose(exposure.data.mean(), 3.94257338e08, rtol=3e-3)
 
 
+@requires_data()
+def test_failure_datasets_maker_spectrum(
+    observations_hess, failure_makers_spectrum, spectrum_dataset
+):
+    makers = DatasetsMaker(failure_makers_spectrum, stack_datasets=True, n_jobs=4)
```

Comment:
Here I would suggest no to call these `failure_xx`, you want to make sure that makers for spectra are handled correctly. The `makers.run` call is not supposed to fail with this correction.